prow/istio-presubmit.sh

```diff
@@ -30,47 +29,9 @@ set -u
 set -x
 set -e
 
-die () {
-  echo "$@"
-  exit -1
-}
+source ${ROOT}/prow/lib.sh
+setup_and_export_git_sha
 
-if [ "${CI:-}" == 'bootstrap' ]; then
-  # Handle prow environment and checkout
-  export USER=Prow
-
-  # Test harness will checkout code to directory $GOPATH/src/github.com/istio/istio
-  # but we depend on being at path $GOPATH/src/istio.io/istio for imports
-  mv ${GOPATH}/src/github.com/istio ${GOPATH}/src/istio.io
-  ROOT=${GOPATH}/src/istio.io/istio
-  cd ${GOPATH}/src/istio.io/istio
-
-  # Use the provided pull head sha, from prow.
-  GIT_SHA="${PULL_PULL_SHA}"
-
-  # check if rewrite history is present
-  PR_BRANCH=$(git show-ref | grep refs/pr | awk '{print $2}')
```

Comment:
You need to keep it. This part is to check if the branch is cloned from old history (before mono-repo change)

Reply:
that s ok, we are well passed the monorepo change.